cmd/ssl-expiry-check/main.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Comcast Cable Communications Management, LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package ssl-expiry-check implements an SSL expiration checker for Kuberhealthy
+// It verifies that a domain's SSL cert is valid, and does not expire in the next 60 days
+
+package main
+
+import (
+	"os"
+
+	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/checkclient"
+	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/ssl_util"
+	log "github.com/sirupsen/logrus"
+)
+
+var domainName string
+var portNum string
+var daysToExpire string
+
+func init() {
+	domainName = os.Getenv("DOMAIN_NAME")
+	if len(domainName) == 0 {
+		log.Errorln("ERROR: The DOMAIN_NAME environment variable has not been set.")
+		return
+	}
+	portNum = os.Getenv("PORT")
+	if len(portNum) == 0 {
+		log.Errorln("ERROR: The PORT environment variable has not been set.")
+		return
+	}
+	daysToExpire = os.Getenv("DAYS")
+	if len(daysToExpire) == 0 {
+		log.Errorln("ERROR: The DAYS environment variable has not been set.")
+		return
+	}
+}
+
+func main() {
+	runExpiry()
+}
+
+func runExpiry() {
```

Comment:
i would say return the error in this function and handle the os.Exit(1) in the main() func

Reply:
also maybe `reportKHSuccess` and `reportKHFailure` in the main? So if there are no errors from running `runExpiry()` --> `reportKHSuccess()` and if there are errors, `reportKHFailure(err)`